torch/_inductor/codegen/wrapper.py

```diff
@@ -298,6 +298,7 @@ def write_header(self):
                 import random
                 import os
                 import tempfile
+                from math import inf, nan
```

Comment:
Note: for the nan case, I didn't find a case to cover it, but I think it will be better to import it.